### ISSUE TITLE
can't connect to a server if other user already connected is using non English characters in the channel name placeholder

### ISSUE THREAD

**Issue by jonjamcam** [elieserdejesus/JamTaba#457]
I can use any other server, but this one says I'm connected, but I never get to get in. Tried for about 15 minutes now...

I tried vst and standalone, restarted the computer, but nothing. Strangely it says in the jamroomwindow  and in ninbot.com that I'm connected.

I enterd via reaninjam as jon and tried TEST for jamtaba

![image](https://cloud.githubusercontent.com/assets/15310433/16134999/03218ed0-33f7-11e6-9d28-8d48f5d99114.png)

EDIT: this could be related with https://github.com/elieserdejesus/JamTaba/issues/294

**Comment by jonjamcam** [comment]
Just tested and it's working now.

**Comment by jonjamcam** [comment]
I think I found a nasty bug. It happened to me twice today. It is this same bug reported here. Jamtaba users can't enter a server when Zenuck joins. It's not about him, obviously, but I noticed he uses protuguese characters in his channel name. It's just a guess, but it may be important. Maybe this characters (and others) are not supported.

![image](https://cloud.githubusercontent.com/assets/15310433/16359126/6efd8776-3aff-11e6-89cf-f2e8d760330e.png)

**Comment by elieserdejesus** [comment]
@jonjamcam , I tested using "Violão/Baixo" as channel name and can't reproduce the bug. I tested in local server and ninbot 2051.

**Comment by jonjamcam** [comment]
I told zenuck just to be sure to use only neutral characters. Let's keep an eye on this one....

>can't reproduce the bug

maybe it's a combination of things. I played today with him and this portuguese letters, but then he disconnected and when he got in again, I was kicked out.

**Comment by jonjamcam** [comment]
@elieserdejesus I did some tests today and I was able to reproduce this bug. First it's not about ninbot:2051; it can happen with any server. The problem happens when someone connects to a server using non english letters (i.e. accents), (who's connection works fine) **and** another jamtaba user tries to connect to the same server. This last user can't connect, but for others he/she appears conected (as shown in the first image here https://github.com/elieserdejesus/JamTaba/issues/457#issue-160772778). For reproducing you'll need two instances of jamtaba and one  of reaninjam.

1.- open one instance of Jamtaba and connect to a server.
2.- open  reaninjam and connect to same server. 

If you use accents on any of the two connections above in the **channel name** (either jamtaba or reaninjam) a third jamtaba user will not be able to connect (independently from the letters this last user puts in the channel placeholder)

3.- Connect using a second instance of Jamtaba.

If you do not use accents in any of the 1 and 2 connections, you will be able to enter the server normally.
